tensorflow_datasets/core/dataset_builder.py

```diff
@@ -135,6 +156,11 @@ class DatasetBuilder(object):
   image, label = features['image'], features['label']
   ```
   """
+  _original_state: Dict[Any, Any]
```

Comment:
Rather than defining annotation here, you should move them in `__init__`

```
def __init__(...):
  self._original_state: Dict[Any, Any] = ...
```

Reply:
It is necessary to define in that way. Otherwise it is giving error as below:
```
 No attribute '_original_state' on DatasetBuilder [attribute-error]
```

Reply:
It should works as per https://www.python.org/dev/peps/pep-0526/#class-and-instance-variable-annotations:

See my other message above

Reply:
Done